runtime/domain.c

```diff
@@ -842,13 +842,13 @@ static void stw_resize_minor_heap_reservation(caml_domain_state* domain,
     caml_gc_log("stw_resize_minor_heap_reservation: "
                 "unreserve_minor_heaps");
 
-    unreserve_minor_heaps();
+    unreserve_minor_heaps_from_stw_single();
     /* new_minor_wsz is page-aligned because caml_norm_minor_heap_size has
        been called to normalize it earlier.
     */
     caml_minor_heap_max_wsz = new_minor_wsz;
     caml_gc_log("stw_resize_minor_heap_reservation: reserve_minor_heaps");
-    reserve_minor_heaps();
+    reserve_minor_heaps_from_stw_single();
     /* The call to [reserve_minor_heaps] makes a new reservation,
```

Comment:
`reserve_minor_heaps`'s name was not changed here

Reply:
Fixed, thanks! (In the logs the un-suffixed names are used in places, but I prefer to keep them as-is as they are shorter and introduce no risk of confusion.)